pkgdb2/api/extras.py

```diff
@@ -216,7 +218,10 @@ def api_bugzilla():
     acls = _bz_acls_cached(name, out_format)
 
     if out_format == 'json':
-        return flask.jsonify(acls)
+        return flask.Response(
+            str(acls).replace("'", '"').replace('None', 'null'),
```

Comment:
Usage of ``str`` here is ambiguous, no?  Perhaps instead you want ``unicode(acls).encode('utf-8')``?